airflow/jobs/base_job.py

```diff
@@ -113,36 +113,15 @@ def __init__(self, executor=None, heartrate=None, *args, **kwargs):
         self.unixname = getuser()
         self.max_tis_per_query: int = conf.getint("scheduler", "max_tis_per_query")
         get_listener_manager().hook.on_starting(component=self)
-        super().__init__(*args, **kwargs)
+        self.job_type = job_runner.get_job_type()
```

Comment:
nit: Why storing it since you can access it using `self.job_runner`. Plus, it seems you do not use it

Reply:
It's defined above as the ORM field in BaseJob. So the whole reason of setting it here is to make sure that the `job_type` is retrieved from runner and stored in the database.

Reply:
I moved it up and added comment about it.